addons/account/wizard/account_payment_register.py

```diff
@@ -657,6 +657,10 @@ def _create_payments(self):
                     for line in batch_result['lines']:
                         new_batches.append({
                             **batch_result,
+                            'payment_values': {
+                                **batch_result['payment_values'],
+                                'payment_type': 'inbound' if line.balance > 0 else 'outbound'
```

Comment:
This line seems weird but I see it's the same than in the get_batch method so I guess it's normal